### PR TITLE
PMM-7: fix docs pipeline with empty grep

### PR DESCRIPTION
**Pull request by michal-kralik**
PMM-7

### CODE REVIEW

File: .github/workflows/release-doc.yml [hunk 1]
```diff
@@ -17,7 +17,7 @@ jobs:
         env:
           release_tag: ${{ github.ref }}
       - name: Convert mkdocs
-        shell: bash
+        shell: bash --noprofile --norc -e {0}
```
Comment (atymchuk) [inline .github/workflows/release-doc.yml:20]:
```suggestion
        shell: bash --noprofile --norc -ex {0}
```
Maybe adding -x (aka `xtrace`) will help troubleshoot?

Comment (michal-kralik) [reply .github/workflows/release-doc.yml]: The `-o pipefail` made the pipeline fail on `grep` if the pattern was not found

**Review by atymchuk — APPROVED** [review]

**Comment by artemgavrilov** [comment]
@michal-kralik But what was the reason of failure?

**Comment by michal-kralik** [comment]
> @michal-kralik But what was the reason of failure?

The flag `-o pipefail` makes the pipeline fail with exit code of the first command which failed. 
By default pipeline returns the exit code of the last command.

In our case `grep` exits with code `1` because it did not find any match.

See https://gist.github.com/mohanpedala/1e2ff5661761d3abd0385e8223e16425#set--o-pipefail for reference

**Pull request merged**